### PR TITLE
Plat 13085 list modules

### PR DESCRIPTION
**Pull request by taylor-cb**
## Ticket

PLAT-13085

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [X] Minor changes

Did you make changes to modules or created a new module?

- [ ] Yes, and have read the [modules updates checklist](https://github.com/crowdbotics/modules/#modules-updates-checklist) documentation and followed the instructions.
- [X] No.

## Changes introduced

Adding modules list command, with some basic search capabilities
More search flags are pending

Search usage:
image

Basic Pagination:
image

## Test and review

_Describe how a reviewer should test your PR_

### CODE REVIEW

**Review by alaisgomes — APPROVED** [review]

**Review by Metalheart88 — APPROVED** [review]

File: index.js
Comment (taylor-cb) [reply index.js:246]: I was just waiting for final approval on filter options before adding help. I'll do it in a separate pr but in the same ticket.

**Review by alaisgomes — APPROVED** [review]

**Pull request merged**
